veriphix/client.py

```diff
@@ -366,3 +515,29 @@ def set_measure_result(self, node: int, result: bool) -> None:
         if self.__client.secret_datas.r:
             result ^= self.__client.secret_datas.r[node]
         self.__client.results[node] = result
+
+
+class TestMeasureMethod(MeasureMethod):
+    def __init__(self, client: Client):
+        self.__client = client
+
+    def get_measurement_description(self, cmd: BaseM) -> Measurement:
+        # Extract secrets from Client
+        r_value = self.__client.secret_datas.r.get(cmd.node, 0)
+        theta_value = self.__client.secret_datas.theta.get(cmd.node, 0)
+        a_N_value = self.__client.secret_datas.a.a_N.get(cmd.node, 0)
+
+        # Blind the angle using the Client's secrets
+        angle = theta_value * np.pi / 4 + np.pi * (r_value + a_N_value)
```

Comment:
With the `blind_angle` introduced above:
```suggestion
        # Blind the angle using the Client's secrets
        angle = blind_angle(self.__client.secret_datas, cmd.node, False)
```